lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4145,6 +4146,34 @@ namespace {
       return false;
     }
 
+    // Inject lifetime annotations selectively for some STL types so we can use
+    // unsafeAddress to avoid copies.
```

Comment:
Do we plan to also add these to libc++? It's still good to have this for compatibility with other STL implementations of course

Reply:
There are some discussions about this but it is not clear yet when would this happen and who would do this work. But as you mentioned, even after we annotate libc++, we still want to inject some annotations strategically to properly support some features with other STL implementations.